Build instructions don't quite work

Starting from a fresh repo, I followed the instructions under "Building" in the readme, but they didn't quite work. The build failed as shown below.

I found that I was able to fix this problem by symlinking the directories under `packages` into the `node_modules` directory, in particular so that `capnp-ts` successfully resolved, but I presume that's not the "right" solution. Note that I haven't used Yarn, Gulp, or Typescript before, but have used npm and node plenty.

    kenton@zero:~/src/capnp-ts:master$ yarn install
    yarn install v0.27.5
    [1/4] Resolving packages...
    [2/4] Fetching packages...
    [3/4] Linking dependencies...
    [4/4] Building fresh packages...
    Done in 2.57s.
    kenton@zero:~/src/capnp-ts:master$ npm run build
    
    > capnp-ts-all@0.0.1 build /home/kenton/src/capnp-ts
    > gulp build
    
    [12:52:17] Using gulpfile ~/src/capnp-ts/gulpfile.js
    [12:52:17] Starting 'build:capnp-ts:lib'...
    [12:52:19] Finished 'build:capnp-ts:lib' after 2.59 s
    [12:52:19] Starting 'build:capnpc-ts:lib'...
    
    5 import * as s from 'capnp-ts/lib/std/schema.capnp';
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    
    packages/capnpc-ts/src/ast-creators.ts(5,20): error TS2307: Cannot find module 'capnp-ts/lib/std/schema.capnp'.
    
    
    6 import {format, pad} from 'capnp-ts/lib/util';
                                ~~~~~~~~~~~~~~~~~~~
    
    packages/capnpc-ts/src/ast-creators.ts(6,27): error TS2307: Cannot find module 'capnp-ts/lib/util'.
    
    
    5 import * as s from 'capnp-ts/lib/std/schema.capnp';
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    
    packages/capnpc-ts/src/code-generator-file-context.ts(5,20): error TS2307: Cannot find module 'capnp-ts/lib/std/schema.capnp'.
    
    ... more similar errors follow ...

--- a/README.md
+++ b/README.md
@@ -32,6 +32,7 @@ This is a TypeScript implementation of the [Cap'n Proto](https://capnproto.org) 
   - [Usage with JavaScript](#usage-with-javascript)
   - [Usage in a Web Browser](#usage-in-a-web-browser)
 - [Building](#building)
+  - [Initial Setup](#initial-setup)
   - [Build Tasks](#build-tasks)
 - [Testing](#testing)
 - [Team](#team)
@@ -128,7 +129,16 @@ Before building the source you will need a few prerequisites:
 
 > [nvm](https://github.com/creationix/nvm) is highly recommended for managing multiple nodejs versions.
 
-Run `yarn install` to install the initial set of build packages before continuing.
+### Initial Setup
+
+Run the following commands to set up the **node_modules** directories for the monorepo and each package:
+
+```shell
+yarn install
+npm run bootstrap
+```
+
+Bootstrap only needs to be run once; run `yarn install` again any time the packages are updated.
 
 ### Build Tasks
 

